### ISSUE TITLE
Make node core modules available as globals in n>

### ISSUE THREAD

**Issue by brigand** [brigand/jellobot#11]
Sample code from devsnek

```js
for (const name of require('repl')._builtinLibs) {
  const setReal = (val) => {
    delete global[name];
    global[name] = val;
  };
  Object.defineProperty(global, name, {
    get: () => {
      const lib = require(name);
      delete global[name];
      Object.defineProperty(global, name, {
        get: () => lib,
        set: setReal,
        configurable: true,
        enumerable: false,
      });
      return lib;
    },
    set: setReal,
    configurable: true,
    enumerable: false,
  });
}
```

**Comment by caub** [comment]
@devsnek

the script above give this behavior:
```js
global.fs=1
console.log(fs)
delete global.fs
console.log(global.fs)
// 1
// undefined
```

This: seems enough to produce that same behavior
```js
for (const name of require('repl')._builtinLibs) {
  Object.defineProperty(global, name, {
    get: () => {
      const lib = require(name);
      delete global[name];
      return lib;
    },
    set: val => {
      delete global[name];
      global[name] = val;
    },
    configurable: true,
    enumerable: false,
  });
}
```
unless I'm missing some corner case

**Comment by brigand** [comment]
Fixed in #14
